coss/discourse/tests/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 
 class DiscourseCategoryFactory(factory.DjangoModelFactory):
-    category_id = 198
+    category_id = 245
```

Comment:
Thanks @hmitsch for this PR. This piece of code here is relevant to the tests and it does not affect the site. The category is controlled through the admin interface. Thanks for taking the time to fix this though.